nimble/host/src/ble_hs_hci.c

```diff
@@ -574,4 +611,15 @@ ble_hs_hci_init(void)
 
     rc = ble_npl_mutex_init(&ble_hs_hci_mutex);
     BLE_HS_DBG_ASSERT_EVAL(rc == 0);
+
+    rc = mem_init_mbuf_pool(ble_hs_hci_frag_data,
+                            &ble_hs_hci_frag_mempool,
+                            &ble_hs_hci_frag_mbuf_pool,
+                            1,
+                            BLE_HS_HCI_FRAG_MEMBLOCK_SIZE,
+                            "ble_hs_hci_frag");
+    BLE_HS_DBG_ASSERT_EVAL(rc == 0);
+
+    rc = os_msys_register(&ble_hs_hci_frag_mbuf_pool);
+    BLE_HS_DBG_ASSERT_EVAL(rc == 0);
```

Comment:
this dedicated pool should not be registered with msys since anyone will be able to grab our dedicated buffer

Reply:
Oops, you're right.  Thanks for catching that.

Reply:
I have removed those last two lines.